reflex/vars.py

```diff
@@ -2003,3 +2003,9 @@ def __call__(self, *args, **kwargs) -> BaseVar:
             The Var returned from calling the function.
         """
         return self.fn(*args, **kwargs)
+
+
+unique_string_var = BaseVar(
+    _var_name="crypto.randomUUID()",
```

Comment:
I was doing some more research into this and apparently it's only available "secure contexts" in some browsers. I think more testing is needed around this, or we might get browser errors in some cases.

Reply:
Right, an alternative may be to use a third-party library like https://www.npmjs.com/package/uuid , and even they advise using `crypto.randomUUID()` if you only need to generate uuids and I'm also very reluctant on the idea of installing a new package just to generate random UUIDs. Another option would be having to do this ourselves, i.e writing a helper function to generate random strings. What do you think?

Reply:
we already have our own function in `state.js`, maybe we can just export and use that?

https://github.com/reflex-dev/reflex/blob/6384c62e51cc354b0c1071c8a7ffa66cabd51a17/reflex/.templates/web/utils/state.js#L43-L57

Reply:
yes, that really should work. Will update the the PR to use this instead

Reply:
fixed